ADO.NET/AdoOauthSample/app.config

```diff
@@ -0,0 +1,23 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+	<!--
+      ConnectionStrings defines how the SampleApp will connect to the server.  Change this to be specific to your environment.
+    -->
+	<connectionStrings>
+		<add name="ConnectionString" connectionString="Host=localhost;Port=5433;Database=VMart;User=dbadmin;Password=;OAuthAccessToken=;" providerName="Vertica.Data" />
+	</connectionStrings>
+	<!--
+      App settings control how the sample queries your database.   You can change the Query and adjust the output by changing MaxColumnWidth.
+    -->
+	<appSettings>
+
+		<add key="Query" value="select node_name, node_state, node_address, is_primary, is_readonly from nodes;" />
+
+		<!-- 
+      Specifies the maximum width that will be used for any column.  Column headers and data values will be large enough for
+      the complete column header, but any data values longer than that will be truncated.   If values are truncated a warning
+      message is printed.  MaxColumnWidth values less than zero disables this limit.
+    -->
+		<add key="MaxColumnWidth" value="15"/>

```

Comment:
This is not used.  It could be if we had common code for the ResultSetPrinter.

Reply:
I'll remove it.